README.md

```diff
@@ -1 +1,37 @@
-# ES6_Awesome_books
\ No newline at end of file
+# ES6_Awesome_books
+Awesome books is a project where we can add the list of books and store it at locally.
+
+## Built With
+
+- HTML5, CSS3, JS
+- ES6
+
+## Live Project
+
+[Click here to view](https://tahseenzahra.github.io/ES6_Awesome_books/)
```

Comment:
Great job @tahseenzahra 🤝

- [ ] [Optional] Kindly make sure your project is well deployed.

Reply:
Thanks for the review :slightly_smiling_face:  ya I think I would work because I checked there was the main branch that was selected so after merging it must work.